packages/block-library/src/gallery/save.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * WordPress dependencies
+ */
+import { RichText } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import { defaultColumnsNumber } from './shared';
+
+export default function	save( { attributes } ) {
+	const { images, columns = defaultColumnsNumber( attributes ), imageCrop, linkTo } = attributes;
```

Comment:
If this function is only used once, why not place it in this file?

Reply:
`defaultColumnsNumber` is used also in `edit.js` and `index.js` at the moment.